### PR TITLE
Updated generate-add-sshkey.md

### PR DESCRIPTION
**Pull request by CarrieBrown**
Updates for new portal

### CODE REVIEW

File: new-docs-jan-2020/generate-add-sshkey.md [hunk 1]
```diff
@@ -90,17 +90,17 @@ To add your public key to the OSG Connect log in server:
 
 1. Go to www.osgconnect.net and sign in with your Globus ID.
```
Comment (ChristinaLK) [inline new-docs-jan-2020/generate-add-sshkey.md]: Should this say CILogin?

**Comment by ChristinaLK** [comment]
Looks good!

**Comment by ChristinaLK** [comment]
Oops, also, maybe we should have a note in the log in section that people will be using the specific login node they've been assigned.

**Comment by CarrieBrown** [comment]
Added info on how to figure out the assigned login node and removed references to `login.osgconnect.net`

**Pull request merged**